cmds/stage

```diff
@@ -145,7 +145,12 @@ stage_repository() {
             *) img_format="file" ;;
         esac
 
-        local img_src_name="${img_src_label}-${machine}.${img_type}"
+        local img_src_name
+        case "${img_src_label}" in
+            control) img_src_name="${img_src_label}.${img_type}" ;;
```

Comment:
This makes backwards compatibility okay since the source-label changes in the template.